### PR TITLE
fix(bug): fix form bug raised by customer

### LINKED ISSUE #458: Contact Form Checkbox Alignment
The checkboxes are not aligned properly

https://main--mammotome--hlxsites.hlx.page/de/de/kontakt

Screenshot 2023-07-12 at 18 04 22

### PR DESCRIPTION
**Pull request by Pareesh**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #458 #438 #426 #437 

Test URLs:
Before: https://main--mammotome--hlxsites.hlx.page/us/en/contact/
After: https://formbug2--mammotome--hlxsites.hlx.page/us/en/contact/
Before: https://main--mammotome--hlxsites.hlx.page/us/en/schedule-a-demo
After: https://formbug2--mammotome--hlxsites.hlx.page/us/en/schedule-a-demo
Before: https://main--mammotome--hlxsites.hlx.page/us/en/submit-a-tech-support-ticket
After: https://formbug2--mammotome--hlxsites.hlx.page/us/en/submit-a-tech-support-ticket
Before: https://main--mammotome--hlxsites.hlx.page/us/en/about-us/innovate-with-mammotome
After: https://formbug2--mammotome--hlxsites.hlx.page/us/en/about-us/innovate-with-mammotome
Before: https://main--mammotome--hlxsites.hlx.page/us/en/about-us/investigator-initiated-study-request
After: https://formbug2--mammotome--hlxsites.hlx.page/us/en/about-us/investigator-initiated-study-request
Before: https://main--mammotome--hlxsites.hlx.page/us/en/products/breast-biopsy-markers/
After: https://formbug2--mammotome--hlxsites.hlx.page/us/en/products/breast-biopsy-markers/

### CODE REVIEW

File: blocks/form/form.js [hunk 1]
```diff
@@ -223,9 +225,11 @@ const createSelect = withFieldWrapper((fd) => {
     ph.setAttribute('disabled', '');
     select.append(ph);
   }
-  fd.Options.split(',').forEach((o) => {
+  const options = fd.Options.split(',');
+  const optionsName = fd['Options Name'] ? fd['Options Name'].split(',') : options;
+  options.forEach((o, index) => {
     const option = document.createElement('option');
-    option.textContent = o.trim();
+    option.textContent = optionsName[index].trim();
```
Comment (Pareesh) [inline blocks/form/form.js:232]: We are still in progress of deciding how to move forward with options list. Country options contain ',' which we used to use as a delimiter.

**Review by solaris007 — APPROVED** [review]

File: blocks/form/form.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ function loadScript(url) {
 }
 
 function constructPayload(form) {
-  const payload = {};
+  const payload = { submissionDate: (new Date()).toUTCString() };
```
Comment (jalagari) [inline blocks/form/form.js:23]: Is it expected UTC string?

File: blocks/form/form.js [hunk 3]
```diff
@@ -373,32 +389,26 @@ async function createForm(formURL) {
...
-  decorateUTM(form);
-  decorateSupScriptInTextBelow(form);
   // eslint-disable-next-line prefer-destructuring
   form.dataset.action = pathname.split('.json')[0];
   form.addEventListener('submit', (e) => {
     e.preventDefault();
     e.submitter.setAttribute('disabled', '');
-    handleSubmit(form, e.submitter.dataset?.redirect);
+    e.submitter.dataset.text = e.submitter.textContent;
+    e.submitter.textContent = 'Please Wait.';
```
Comment (jalagari) [inline blocks/form/form.js:398]: We need to localize this string mostly using placeholder mechansim.

File: blocks/form/form.js (hunk 1 above)
Comment (jalagari) [reply blocks/form/form.js:232]: Lets processed with approach #1 i.e. endpoint

File: blocks/form/form.js (hunk 2 above)
Comment (Pareesh) [reply blocks/form/form.js:23]: I added this based on my understanding, I have asked the customer https://adobe-dx-support.slack.com/archives/C04T6MD0TDM/p1689250277874679 
Thank you

**Review by jalagari — APPROVED** [review]

**Pull request merged**